src/rapsV2/actions/claimTransactionClaimableAction.ts

```diff
@@ -7,39 +7,37 @@ import { NewTransaction } from '@/entities';
 import { chainsName } from '@/chains';
 
 export async function claimTransactionClaimable({ parameters, wallet }: ActionProps<'claimTransactionClaimableAction'>) {
-  // will uncomment actual logic in follow-up PR, don't worry about it for now
-  return { nonce: undefined, hash: undefined };
+  const { claimTx, asset } = parameters;
 
-  // const { claimTx } = parameters;
+  const provider = getProvider({ chainId: claimTx.chainId });
+  const result = await sendTransaction({ transaction: claimTx, existingWallet: wallet, provider });
 
-  // const provider = getProvider({ chainId: claimTx.chainId });
-  // const result = await sendTransaction({ transaction: claimTx, existingWallet: wallet, provider });
+  if (!result?.result || !!result.error || !result.result.hash) {
+    throw new RainbowError('[CLAIM-TRANSACTION-CLAIMABLE]: failed to execute claim transaction');
+  }
 
-  // if (!result?.result || !!result.error || !result.result.hash) {
-  //   throw new RainbowError('[CLAIM-TRANSACTION-CLAIMABLE]: failed to execute claim transaction');
-  // }
+  const transaction = {
+    amount: '0x0',
+    gasLimit: result.result.gasLimit,
+    from: result.result.from ?? null,
+    to: result.result.to ?? null,
```

Comment:
should we fail if to/from are null? That basically is sending code to a new contract I think